src/Bridge/Symfony/Routing/RouteNameResolverInterface.php

```diff
@@ -32,5 +32,5 @@
      *
      * @return string
      */
-    public function getRouteName(string $resourceClass, $operationType): string;
+    public function getRouteName(string $resourceClass, $operationType, array $context = []): string;
```

Comment:
I think that this is a breaking change. Should I avoid changing this interface? I need more informations when retrieving the route name for a subresource because multiple routes may find a wrong match on the same resource class. It has to take into consideration the subresource tree. See `isSameSubresource` in `RouteNameResolver`.